templates/careers/all.html

```diff
@@ -0,0 +1,44 @@
+{% extends '/careers/careers_base.html' %}
+
+{% block hero %}
+  <section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom u-extra-space">
+    <div class="p-strip--suru-background">
+      <div class="row">
+        <div class="col-6">
+          <h1>Admin</h1>
+        </div>
+      </div>
+    </div>
+  </section>
+{% endblock %}
+
+{% block overview %}
+  <div class="p-strip is-shallow u-extra-space tab-content" id="overview">
+    <div class="row">
+      <div class="col-12">
+        <p>
+          <strong>Canonical is a completely new kind of organisation - almost entirely distributed, we are a global team of technology leaders who collaborate online to enable the transformation of enterprise software to open source. We have a few office locations, but almost everybody works from home. Teams get together every few months at locations around the world to plan and coordinate their projects.</strong>
+        </p>
+        <p>
+          All of this means that administration is central to the effective running of the company. Hundreds of distributed team members count on effective processes to support their travel and operations. Our admin team run events, control expenses, manage travel and coordinate meetings. We are trusted to hold the global team accountable to the company policies. We ensure that the business operates smoothly and we have a significant impact on the quality of the experience that people have at the company.
+        </p>
+        <p>
+          We are a multi-skilled and multi-cultural team. We encourage new members of the team to develop skills across the entire range of admin challenges so that we can help one another as needed and bring our joint insights to bear on continuous improvement. Our team is largely based in offices in Austin, Boston, London, Taipei and Beijing, but we offer some flexibility to work from home. We often run company events around the world in person, so you need the latitude to be in a completely different timezone for up to two weeks at a time, three or four times a year.
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">
+        <hr />
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">
+        <p class="p-muted-heading">
+          <span style="text-transform: capitalize">Austin&nbsp;&#124;&nbsp;Beijing&nbsp;&#124;&nbsp;Boston&nbsp;&#124;&nbsp;London&nbsp;&#124;&nbsp;Taipei</span>
```

Comment:
Are these characters URL-encoded for a reason? 'Cos you're trying to explicitly prevent wrapping?

Reply:
Yes, I don't want them to wrap.